src/client/terminals/codeExecution/helper.ts

```diff
@@ -33,10 +30,9 @@ export class CodeExecutionHelper implements ICodeExecutionHelper {
             // On windows cr is not handled well by python when passing in/out via stdin/stdout.
             // So just remove cr from the input.
             code = code.replace(new RegExp('\\r', 'g'), '');
-            const pythonPath = this.configurationService.getSettings(resource).pythonPath;
             const args = [path.join(EXTENSION_ROOT_DIR, 'pythonFiles', 'normalizeForInterpreter.py'), code];
-            const processService = await this.processServiceFactory.create(resource);
-            const proc = await processService.exec(pythonPath, args, { throwOnStdErr: true });
+            const processService = await this.pythonServiceFactory.create({ resource: resource });
```

Comment:
You can leverage [object property shorthands](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015) so that you don't have to repeat the object property name:
```suggestion
            const processService = await this.pythonServiceFactory.create({ resource });
```
I like the conciseness, but some find it less readable, so it's up to you.